frontend/app/src/entities/diff/node-diff/utils.tsx

```diff
@@ -62,6 +64,38 @@ export const DiffRow = ({
   iconClassName,
   status,
 }: DiffRowProps) => {
+  if (!children) {
```

Comment:
`!children` makes me think we should have instead another componen that wrap both cases